cirq/ops/common_channels.py

```diff
@@ -50,90 +60,156 @@ def __init__(self, p_x: float, p_y: float, p_z: float) -> None:
             $$
```

Comment:
This can be removed now since a more general formula is added above.

Reply:
Done.